index.md

```diff
@@ -18,10 +18,13 @@ These examples use the `talk` action, producing text-to-speech output.
 * [`dtmf-input.json`](dtmf-input.json) if you want to get the user to enter some digits with their telephone keypad
 * [`transfer.json`](transfer.json) to share a message when a `transfer` action has finished
 
-## Audio
+## Record
 
 * [`record.json`](record.json) will record your call. Take a look at the [code snippets](https://developer.nexmo.com/voice/voice-api/code-snippets/download-a-recording) for examples of how to authenticate and download the finished recording after the call.
 
+## Audio
+* [`silent-loop.json`](silent-loop.json) will play audio file specified in the provided url.
```

Comment:
Maybe something a bit clearer in the one-liner description? Suggestion: "will play an audio file in a loop specified in the provided URL." ?

Reply:
"streams an mp3 file of silence, on a loop. Useful for keeping a call active while waiting for another event to occur" ... something like this?